modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json

```diff
@@ -2,423 +2,3531 @@
+			"productExternalReferenceCode": "RAYAP-001",
+			"r_userToSalesGoals_userId": 20125
+		},
+		{
+			"finalReferenceDate": "2022-02-28T00:00:00Z",
+			"goalValue": 15500,
+			"initialReferenceDate": "2022-02-01T00:00:00Z",
+			"productExternalReferenceCode": "RAYAP-001",
+			"r_userToSalesGoals_userId": 20125
+		},
+		{
+			"finalReferenceDate": "2022-03-30T00:00:00Z",
+			"goalValue": 15500,
+			"initialReferenceDate": "2022-03-01T00:00:00Z",
+			"productExternalReferenceCode": "RAYAP-001",
+			"r_userToSalesGoals_userId": 20125
+		},
+		{
+			"finalReferenceDate": "2022-04-30T00:00:00Z",
+			"goalValue": 15500,
+			"initialReferenceDate": "2022-04-01T00:00:00Z",
+			"productExternalReferenceCode": "RAYAP-001",
+			"r_userToSalesGoals_userId": 20125
+		},
+		{
+			"finalReferenceDate": "2022-05-30T00:00:00Z",
+			"goalValue": 15500,
+			"initialReferenceDate": "2022-05-01T00:00:00Z",
+			"productExternalReferenceCode": "RAYAP-001",
+			"r_userToSalesGoals_userId": 20125
+		},
+		{
+			"finalReferenceDate": "2022-06-30T00:00:00Z",
+			"goalValue": 10500,
+			"initialReferenceDate": "2022-06-01T00:00:00Z",
+			"productExternalReferenceCode": "RAYAP-001",
+			"r_userToSalesGoals_userId": 20125
+		},
+		{
+			"finalReferenceDate": "2022-07-30T00:00:00Z",
+			"goalValue": 15500,
+			"initialReferenceDate": "2022-07-01T00:00:00Z",
+			"productExternalReferenceCode": "RAYAP-001",
+			"r_userToSalesGoals_userId": 20125
+		},
+		{
+			"finalReferenceDate": "2022-08-31T00:00:00Z",
+			"goalValue": 12500,
+			"initialReferenceDate": "2022-08-01T00:00:00Z",
+			"productExternalReferenceCode": "RAYAP-001",
+			"r_userToSalesGoals_userId": 20125
+		},
+		{
+			"finalReferenceDate": "2022-09-30T00:00:00Z",
+			"goalValue": "16500",
+			"initialReferenceDate": "2022-09-01T00:00:00Z",
+			"productExternalReferenceCode": "RAYAP-001",
+			"r_userToSalesGoals_userId": 20125
+		},
+		{
+			"finalReferenceDate": "2022-10-31T00:00:00Z",
+			"goalValue": 15500,
+			"initialReferenceDate": "2022-10-01T00:00:00Z",
+			"productExternalReferenceCode": "RAYAP-001",
+			"r_userToSalesGoals_userId": 20125
+		},
+		{
+			"finalReferenceDate": "2022-11-30T00:00:00Z",
+			"goalValue": "16500",
+			"initialReferenceDate": "2022-11-01T00:00:00",
+			"productExternalReferenceCode": "RAYAP-001",
+			"r_userToSalesGoals_userId": 20125
+		},
+		{
+			"finalReferenceDate": "2022-12-31T00:00:00Z",
+			"goalValue": 20500,
+			"initialReferenceDate": "2022-12-01T00:00:00",
+			"productExternalReferenceCode": "RAYAP-001",
+			"r_userToSalesGoals_userId": 20125
+		},
+		{
+			"finalReferenceDate": "2022-12-31T00:00:00Z",
+			"goalValue": 20500,
+			"initialReferenceDate": "2022-12-01T00:00:00",
+			"productExternalReferenceCode": "RAYA001",
+			"r_userToSalesGoals_userId": 20125
+		},
+		{
+			"finalReferenceDate": "2022-11-31T00:00:00Z",
+			"goalValue": 15500,
+			"initialReferenceDate": "2022-11-01T00:00:00",
+			"productExternalReferenceCode": "RAYA001",
```

Comment:
Hey,  you forgot to fix it. Please change it to "RAY001". Same for the others.

Reply:
Done